content/docs/custom-stacks/_index.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Custom stacks"
+weight=4
+creatordisplayname = "Andrew Meyer"
+creatoremail = "ameyer@pivotal.io"
+lastmodifierdisplayname = "Andrew Meyer"
+lastmodifieremail = "ameyer@pivotal.io"
++++
+
+A [stack](/docs/using-pack/managing-stacks) provides the buildpack lifecycle with build- and run-time environments (given by a
+build image and a run image). If you're using the `pack` CLI, running `pack suggest-stacks` will display a list of recommended
+stacks that can be used when running `pack create-builder`, along with each stack's associated build and run images.
+
+To create a custom stack for use with `pack create-builder`, the following requirements must be met for the build and run images.
+
+### The stack must be given an identifier
+
+Each image must contain a label called `io.buildpacks.stack.id` with an identifier for the stack (e.g. `io.buildpacks.stacks.bionic`).
+The stack identifier implies ABI (application-binary interface) compatibility with other stacks of that same identifier. For instance,
```

Comment:
Is `ABI (application-binary interface)` necessary here?  It might just be me but it confuses me more than helps.

Reply:
Also, this seems like a good place for a  "Note" blockquote.

Reply:
I might just link it, like we do in the spec